File: src/test/java/CalculatorTest.java
import org.junit.jupiter.api.Test;

public class CalculatorTest {


    @Test
    public void add() {
        Calculator calculator = new Calculator();
        System.out.println(calculator.add(3, 4));

    }
    @Test
    public void subtract() {
        Calculator calculator = new Calculator();

        System.out.println(calculator.subtract(5, 4));
    }

    @Test
    public void multiply() {
        Calculator calculator = new Calculator();

        System.out.println(calculator.multiply(2, 6));
    }

    @Test
    public void divide() {
        Calculator calculator = new Calculator();

        System.out.println(calculator.divide(8, 4));
    }

}

File: src/test/java/calculator/CalculatorTest.kt
package calculator

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

class CalculatorTest {
    private var calculator: Calculator? = null
    @BeforeEach
    fun setup() {
        calculator = Calculator()
    }

    @Test
    fun add() {
        assertEquals(9, calculator!!.add(6, 3))
    }

    @Test
    fun subtract() {
        assertEquals(3, calculator!!.subtract(6, 3))
    }

    @Test
    fun multiply() {
        assertEquals(12, calculator!!.multiply(2, 6))
    }

    @Test
    fun divide() {
        assertEquals(2, calculator!!.divide(8, 4))
    }
}